expertise/create_dataset.py

```diff
@@ -54,13 +54,32 @@ def get_paper_notes(self, author_id, dataset_params):
         notes_v2 = list(openreview.tools.iterget_notes(self.openreview_client_v2, content={'authorids': author_id}))
         return notes_v1 + notes_v2
 
+    def deduplicate_publications(self, publications):
+        deduplicated = []
+
+        # Build index of pub.original
+        original_ids = [getattr(pub, 'original', None) for pub in publications]
+
+        for pub in publications:
+            note_id, original_id = pub.id, getattr(pub, 'original', None)
+            # If original note, but blind already exists, skip the original note
+            if original_id is None and note_id in original_ids:
+                continue
```

Comment:
I would prefer to avoid the `continue` sentence and evaluate when we need to keep the publications, my suggestion is:

```
if note not in original_ids:
     deduplicated.append(pub)
```

Only notes that are not original notes should be used as a publication.